lib/commands/device/index.ts

```diff
@@ -169,6 +170,7 @@ export default class DeviceCmd extends Command {
 				'is_online',
 				'ip_address',
 				'mac_address',
+				'public_address',
```

Comment:
Nitpick: Maybe move this one line up (between `ip_address` and `mac_address`), so that the private and public IP addresses are displayed next to each other?